src/main/java/mate/academy/lib/ConnectionUtil.java

```diff
@@ -0,0 +1,29 @@
+package mate.academy.lib;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/test";
+    private static final String USER = "root";
+    private static final String PASSWORD = "qwerty123";
```

Comment:
Storing database credentials directly in the code is a bad practice due to security reasons. It's better to use environment variables or configuration files that are not included in the version control.